### PR TITLE
Quality Checks

### PR DESCRIPTION
**Pull request by srevinsaju**
Moved from #31

### CODE REVIEW

**Comment by srevinsaju** [comment]
@tchx84 removed the redundant commits and add `qa-requirements.txt`

File: qa/test.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# stop on error, and echo commands to stdout
+set -ex
+
+black --check src
```
Comment (tchx84) [inline qa/test.sh:7]: Still called `qa/test.sh` vs `qa,sh` :P

File: qa-requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+black
+pyflakes
```
Comment (tchx84) [inline qa-requirements.txt:2]: specify the versions with ==x.x.x please!

File: .flake8 [hunk 3]
```diff
@@ -0,0 +1,4 @@
+[flake8]
+exclude = .git,__pycache__,build,dist,build-aux
+ignore = E402,E722 
+max-line-length = 119
```
Comment (tchx84) [inline .flake8:4]: leave this to black.

Comment (srevinsaju) [reply .flake8:4]: wow, ok

**Comment by tchx84** [comment]
moved to #41

**Pull request closed without merging**